timeless/views.py

```diff
@@ -329,6 +323,24 @@ def get(self, object_id):
                 return jsonify(response), HTTPStatus.OK
             abort(HTTPStatus.NOT_FOUND)
 
+        def post(self, payload):
+            """Fake response on post method."""
+            return jsonify(payload), HTTPStatus.OK
+
+        def put(self, payload):
+            """Fake response on put method."""
+            return jsonify(payload), HTTPStatus.OK
+
+        def delete(self, object_id):
+            """Fake response on delete method."""
+            if object_id == 5:
+                response = {
+                        "some_id": 5,
+                        "some_attr": "attr"
```

Comment:
@soin08 indentation here, these json attributes should be intendet with 4 spaces less

Reply:
Fixed